ui-particles-angular/projects/ui-particles-angular/src/lib/gio-form-json-schema/testing-json-schema/endpoints.ts

```diff
@@ -27,7 +27,7 @@ export const endpointsGetResponse = [
                                  }\n                                    }\n                                ]\n                            }\n                        },\n                        "keyContent": {\n                            "type": "string",\n                            "title": "Private key",\n                            "x-schema-form": {\n                                "type": "text",\n                                "hidden": [\n                                    {\n                                        "$neq": {\n                                            "ssl.keyStore.type": "PEM"\n                                        }\n                                    }\n                                ]\n                            }\n                        }\n                    },\n                    "oneOf": [\n                        {\n                            "properties": { "type": { "const": "" } }\n                        },\n                        {\n                            "properties": { "type": { "const": "PEM" } },\n                            "required": ["certContent", "keyContent"]\n                        },\n                        {\n                            "properties": { "type": { "const": "PEM" } },\n                            "required": ["certPath", "keyPath"]\n                        },\n                        {\n                            "properties": { "type": { "const": "PEM" } },\n                            "required": ["certContent", "keyPath"]\n                        },\n                        {\n                            "properties": { "type": { "const": "PEM" } },\n                            "required": ["certPath", "keyContent"]\n                        },\n                        {\n                            "properties": { "type": { "pattern": "JKS|PKCS12" } },\n                            "required": ["content", "password"]\n                        },\n                        {\n                            "properties": { "type": { "pattern": "JKS|PKCS12" } },\n                            "required": ["path", "password"]\n                        }\n                    ]\n                }\n            }\n        }\n    },\n    "required": ["target"],\n    "additionalProperties": false,\n    "x-schema-form": {\n        "errors": {\n            "proxy": {\n                "oneOf": "The host and port are <span class=\\"error\\">required</span>"\n            },\n            "ssl": {\n                "trustStore": {\n                    "oneOf": "A path or a content is <span class=\\"error\\">required</span> - for JKS and PKCS#12 a password is also <span class=\\"error\\">required</span>"\n                },\n                "keyStore": {\n                    "oneOf": "Paths or contents are <span class=\\"error\\">required</span> - for JKS and PKCS#12, a password is also <span class=\\"error\\">required</span>."\n                }\n            }\n        }\n    }\n}\n',
```

Comment:
Maybe the file header needs to be updated ?

Reply:
ha no, we have to revert, it's an attempt with a external validator. and the current schema is not valid with last JsonChema standard